pkg/scheduler/apis/config/legacy_types.go

```diff
@@ -172,47 +170,3 @@ type ExtenderTLSConfig struct {
 	// CAData takes precedence over CAFile
 	CAData []byte
 }
-
-// Extender holds the parameters used to communicate with the extender. If a verb is unspecified/empty,
-// it is assumed that the extender chose not to provide that extension.
-type Extender struct {
```

Comment:
should ExtenderManagedResource and ExtenderTLSConfig be moved out of legacy_types.go as well if we're referencing them in the new v1beta1 config?

Reply:
Good point